astroplan/scheduling.py

```diff
@@ -710,10 +710,8 @@ def _get_filled_indices(self, times):
                                block in self.schedule.scheduled_blocks if
                                isinstance(block, ObservingBlock)])
         for start_end in pre_filled:
-            filled = np.where((start_end[0] < times) & (times < start_end[1]))
-            if len(filled[0]) > 0:
-                is_open_time[filled[0]] = False
-                is_open_time[min(filled[0]) - 1] = False
+            filled = np.where((start_end[0]-0.5*u.second < times) & (times < start_end[1]))
```

Comment:
```suggestion
            if np.diff(start_end) <= np.min(np.diff(times)):
                warnings.warn(
                    "Unexpected behavior may occur when the time "
                    f"resolution ({np.min(np.diff(times))}) "
                    "is not smaller than the block duration "
                    f"({np.diff(start_end)}).", AstroplanWarning
                )
            filled = np.where((start_end[0]-0.5*u.second < times) & (times < start_end[1]))
```

If you accept this change, you'll also need to add the following import to this file:
```python
from .exceptions import AstroplanWarning
```